alf/bin/train_test.py

```diff
@@ -34,17 +36,22 @@ def run_and_stream(cmd, cwd):
         cwd (str): working directory for the process
     """
     logging.info("Running %s", " ".join(cmd))
+
+    logger = logging.ABSLLogger('')
```

Comment:
instead of changing the code here, can we simply redirect stdout and stderr to stderr at build.sh?
```sh
python3 -m unittest discover -p "*_test.py" -v 1>&2
```
If not, we need some comment in the code to explain the reason of doing all these stuff so that future readers can understand it.

Reply:
yes, it can be done with`python3 -m unittest discover -p "*_test.py" -v 1>&2`  , 
but there is a  potential problem  [Log length exceeded 4 MB](https://docs.travis-ci.com/user/common-build-problems/#log_length_exceeded) if we log for all stdout  (now the log file is about 3.2MB)

Reply:
I see. Then please add some comments here.

Reply:
added